tests/test_sentinel2_processing_commands.py

```diff
@@ -95,6 +96,45 @@ def test_download_import_commands(self):
 
         # self.assertTrue(len(result) == 2)
 
+    def test_download_import_commands_noquery(self):
+        gsqi = Sentinel2Processing(product_id="S2A_MSIL1C_20170212T104141_N0204_R008_T31TGJ_20170212T104138",
+                                   bands=["B12", "B08"], download_cache="/tmp",
```

Comment:
Is the currently hardcoded `download_cache="/tmp"` a good idea?
I'd prefer it to be set via config file or ENV variable (whatever is better).

Reply:
Note that this test does not actually download anything. It is merely a test whether the `importer`- module is interpreted correctly and leads to the correct download/import commands. However, it is indeed nicer to make the temporary directory configurable. Since no download actually happens, it is now a class attribute.